### ISSUE TITLE
Compatibility of HPE FlexFabric 10Gb 2-port 534FLB Adapter with coreos

### ISSUE THREAD

**Issue by NiraliSupe** [coreos/bugs#2367]
# Issue Report #

## Bug ##

For Fibre channel storage (FCoE), I am using HP FlexFabric 10Gb 2-Port 534FLB Adapter. It is a converged network Broadcom adapter. 
I opened discussion on CoreOS user group about the compatibility of this adapter with CoreOS and found out that currently, CoreOS don't have the kernel driver that supports FCoE offload (config option SCSI_BNX2X_FCOE).
**Link to the discussion:** https://groups.google.com/forum/#!topic/coreos-user/5v0FPUUd2to 
**Output of `lspci -v` command:** [lspci.txt](https://github.com/coreos/bugs/files/1742631/lspci.txt)

### Container Linux Version ###

```
cat /etc/os-release
PRETTY_NAME="Debian GNU/Linux 8 (jessie)"
NAME="Debian GNU/Linux"
VERSION_ID="8"
VERSION="8 (jessie)"
ID=debian
HOME_URL="http://www.debian.org/"
SUPPORT_URL="http://www.debian.org/support"
BUG_REPORT_URL="https://bugs.debian.org/"
```
```
uname -r
4.14.16-coreos
```

### Environment ###

Baremetal

**Comment by cgwalters** [comment]
FWIW I just checked that this is enabled at least in the Fedora kernel:

```
$ rpm -q kernel
kernel-4.15.9-300.fc27.x86_64
$ grep SCSI_BNX2X_FCOE /usr/lib/modules/4.15.9-300.fc27.x86_64/config 
CONFIG_SCSI_BNX2X_FCOE=m
```

**Comment by bgilbert** [comment]
This should be fixed in alpha 1786.0.1, due shortly.  Thanks for reporting.
